src/initialize.js

```diff
@@ -238,6 +239,7 @@ function applyOverrideHandlers(overrides) {
 export async function initialize({
   loggingService = NewRelicLoggingService,
   analyticsService = SegmentAnalyticsService,
+  googleAnalyticsService = GoogleAnalyticsService,
```

Comment:
Please update js docstring above to include the new param `googleAnalyticsService`